package.json

```diff
@@ -45,11 +47,12 @@
     "fs-extra": "^8.1.0",
     "jest": "^26.4.2",
     "jolocom-lib": "^5.2.1",
+    "log4js": "^6.3.0",
     "mockdate": "^2.0.2",
     "node-fetch": "^2.6.0",
     "prettier": "^1.18.2",
     "regenerator-runtime": "^0.13.5",
-    "sqlite3": "^4",
+    "sqlite3": "^5.0.2",
```

Comment:
why this version bump in this PR?

Reply:
For some reason yarn bumped version on adding other dependency, will remove this change

Reply:
Removed from PR